### ISSUE TITLE
Beta Navigation child items not keyboard accessible

### ISSUE THREAD

**Issue by jds-tag** [w3c/w3c-website#245]
**Describe the issue**
As a W3C site user I want to be able to tab through all menu items using a keyboard.

**To reproduce**
Tab through the menu, open a menu, continue tabbing. You are brought to the next parent menu item and cannot access any children in the dropdown using a keyboard only.

**Expected behavior**
After a dropdown menu is opened and I hit tab, I'm brought to the first child menu item in the dropdown.

**Comment by iadawn** [comment]
Hi @jds-tag, could you provide your OS and browser details please? Also, are you using any assistive technology?

I am using Firefox on Mac OSX and I can't replicate the problem you describe. 

Much appreciated, Kevin

**Comment by jds-tag** [comment]
This wasn't working yesterday in Chrome on Mac OS but it's fixed today!

**Comment by iadawn** [comment]
I have a love/hate relationship with problems that fix themselves. I will take a look but happy for it to stay closed if it has vanished.

**Comment by jds-tag** [comment]
I don't see any use in keeping this open if I can't repro the issue. This is tabbing exactly as I would expect a menu to, so I think your time is better served on other issues. Thank you both for responding and for everything you do as an org!
